overview.dd

```diff
@@ -1044,23 +1067,25 @@ $(SECTION2 Sample D Program (sieve.d),
 
 import std.stdio;
 
-bool[8191] flags;
-
-int main()
+void main()
 {
-    int i, count, prime, k, iter;
+    size_t i, count, prime, k;
```

Comment:
Move `k` and `prime` to where they are used, declaring all your variables at the top of the function is a C-ism